.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+---
+
+### Is your feature request related to a problem? Please describe.
+
+_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_
+
+### Describe the solution you'd like
+
+_A clear and concise description of what you want to happen._
+
+### Describe alternatives you've considered
+
+_A clear and concise description of any alternative solutions or features you've considered._
+
+### Additional context
+
+_Add any other context or screenshots about the feature request here._
```

Comment:
I like the idea of having a feature request template.

But what about the "Ideas" category?

* <https://github.com/netblue30/firejail/discussions/categories/ideas>

When would a feature request be posted using the template vs in a new
discussion?

Reply:
Good points. Spontaneous I would say use "Ideas" if you have an idea and if we come to something more concrete (i.e. how a new option should integrate into profiles, ...) a detailed feature request is written.

Reply:
@rusty-snake [on Sep 16](https://github.com/netblue30/firejail/pull/4515#discussion_r709796487):

> Good points. Spontaneous I would say use "Ideas" if you have an idea and if
> we come to something more concrete (i.e. how a new option should integrate
> into profiles, ...) a detailed feature request is written.

I thought the same.  My worry was mostly that even without a template people
sometimes already open issues for less detailed feature requests instead of
using discussions.  But I guess we can convert to a discussion in those cases,
so nevermind.

Reply:
Maybe we should add a not to feature_request.md (in a later PR) or to CONTRIBUTING.md.

Where we are already at CONTRIBUTING.md, this needs to be updated too. Especially for discussions.